CriteoPublisherSdk/Sources/Criteo.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  Criteo.swift
+//  CriteoPublisherSdk
+//
+//  Copyright © 2018-2020 Criteo. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+
+@objc extension Criteo {
+  public func validateIntegration() {
+    let skanInfo = SkanInfo()
+    if !skanInfo.hasCriteoId {
+      print(
```

Comment:
I remember that there is a logger macro for warnings in the project. Can't you use it ?

Reply:
Macros does not exist in swift ;p

Reply:
This will be reworked anyway while we address the Logging topic. Which current Macros aren't fitting.